diff --git a/src/sync.js b/src/sync.js
index e4005ee652579a49865a9dae761e51b9d28e7a62..5908bd9dd3e30fb1197b61849c96132d92225239 100644
--- a/src/sync.js
+++ b/src/sync.js
@@ -196,7 +196,7 @@ const Sync = async ({ ipfs, log, events, onSynced, start, timeout }) => {
         } catch (e) {
           console.error(e)
           peers.delete(peerId)
-          if (e.code === 'ERR_UNSUPPORTED_PROTOCOL') {
+          if (e.name === 'UnsupportedProtocolError') {
             // Skip peer, they don't have this database currently
           } else {
             events.emit('error', e)
